modules/components/partials/examples/outputs/redpanda.adoc

```diff
@@ -0,0 +1,37 @@
+// This content is autogenerated. Do not edit manually.
+
+== Examples
+
+=== Simple Common Output
+
+Data is generated and written to a topic bar, targetting the cluster configured within the redpanda block at the bottom. This is useful as it allows us to configure TLS and SASL only once for potentially multiple inputs and outputs.
```

Comment:
```suggestion
Data is generated and written to a topic `bar`, targeting the cluster configured within the `redpanda` block at the bottom. This it allows us to configure TLS and SASL only once for potentially multiple inputs and outputs.
```
Just a suggestion, wasn't able to find the preview for this so not sure what the context is